src/test/java/core/basesyntax/services/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.services.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String INPUT_PATH = "src/test/java/testresources/testfruits-log.csv";
+    private static final String EMPTY_FILE = "src/test/java/testresources/empty_fileTest.csv";
+    private static final String INVALID_PATH_EXAMPLE = "123123123123asd";
+    private static FileReader fileReader;
+
+    @Before
+    public void setUp() {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test
+    public void file_path_isValid_ok() {
```

Comment:
all method names in all tests should look like:

methodUnderTesting_condition_result